target/linux/ath79/image/nand.mk

```diff
@@ -43,6 +43,22 @@ define Device/aerohive_hiveap-121
 endef
 TARGET_DEVICES += aerohive_hiveap-121
 
+define Device/domywifi_dw33d
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := DomyWifi
+  DEVICE_MODEL := DW33D
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage kmod-usb-ledtrig-usbport kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  KERNEL_SIZE := 5120k
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  SUPPORTED_DEVICES += dw33d
```

Comment:
Your version is not compatible with dw33d support in ar71xx at all. This SUPPORTED_DEVICES line should be dropped.